packages/pn-pa-webapp/src/App.tsx

```diff
@@ -159,6 +160,10 @@ const App = () => {
       ),
   });
 
+  useEffect(() =>{
```

Comment:
Facendo delle prove, al primo login si ha un errore in quanto viene chiamata la tos senza essere autenticato.
Non si può spostare la getTosApproval nel TosGuard? Se si applicare lo stesso cambiamento a persona fisica